NOpenType/Transformation.cs

```diff
@@ -31,5 +31,13 @@ public Point<short> Transform(Point<short> p)
             var y = (((long)_m10 * px + (long)_m11 * py) >> 14) + _dy;
             return new Point<short>((short)x, (short)y);
         }
+        public Point<short> TransformNormal(Point<short> p)
+        {
+            var px = (long)p.X;
+            var py = (long)p.Y;
+            var x = (((long)_m00 * px + (long)_m10 * py) >> 14) + _dx;
+            var y = (((long)_m01 * px + (long)_m11 * py) >> 14) + _dy;
+            return new Point<short>((short)x, (short)y);
+        }
```

Comment:
This is the only place `transform` was called. Just change the function instead of adding a new one. Just add a comment at top like so

```c#
// WinterDev: The rotation part seems to be transposed.
```